static/js/message_edit.js

```diff
@@ -351,9 +353,21 @@ function edit_message(row, raw_content) {
         message.is_stream && settings_data.user_can_move_messages_between_streams();
     // current message's stream has been already been added and selected in handlebar
     const available_streams = show_edit_stream
-        ? stream_data.subscribed_subs().filter((s) => s.stream_id !== message.stream_id)
+        ? stream_data.subscribed_subs().map((stream) => ({
+              name: stream.name,
+              value: stream.stream_id.toString(),
+          }))
         : null;
 
+    const opts = {
+        widget_name: "select_stream",
+        data: available_streams,
+        default_text: $t({defaultMessage: "No streams"}),
+        include_current_item: false,
```

Comment:
I think we want `include_current_item: true`, because you can change your mind after selecting something and before saving, and want to reset the stream back how it was..